server/amp/helpers/infinite-scroll.js

```diff
@@ -60,7 +61,8 @@ class InfiniteScrollAmp {
     if (!itemsToTake || !storyId)
       return new Error("Required params for getInitialInlineConfig missing");
     const collection = await this.client.getCollectionBySlug(this.collSlug);
-    if (!collection) return null;
+    if (!collection || (collection.items && !collection.items.length))
```

Comment:
`client.getCollectionBySlug` returns `null` if collection doesn't exist and returns 
```
{
  'updated-at': 1616144104833,
  'collection-cache-keys': [ 'c/1363/117898' ],
  slug: 'empty-collection',
  fallback: false,
  name: 'Empty Collection',
  'data-source': 'manual',
  automated: false,
  template: 'default',
  rules: {},
  summary: "Empty Collection for testing - pls don't add stuff to this collection",
  id: 117898,
  'total-count': 0,
  'collection-date': null,
  items: [],
  'created-at': 1616144104785,
  metadata: {}
}
```
for an empty collection.
this function should return null if it's empty collection or if coll doesn't exist, so making this change